docs/README.md

```diff
@@ -129,6 +130,16 @@ In production, this application relies on wikidata.org's OAuth capabilities in o
 
 1. As soon as you receive the email from the WMF team that your consumer is approved, you may start testing your application by logging in through your local instance's home page.
 
+## PHP Linting <a id="phpcs"></a>
+
+The application uses `phpcs` to detect code format violations.
+
+To run phpcs: `./vendor/bin/phpcs`
```

Comment:
This is great! Thanks for updating the docs.

I think it might be easier if we also added a `lint` script to `composer.json` and then use that one in our GH action and the docs instead of calling the phpcs binary directly. That way we will be free to switch tools if we see the need to do so, without having to update everything again.

Also, this would mean we would be able to provide a more consistent development experience by documenting that commands `sail composer run lint` and `sail composer run fix` here instead.

Reply:
Very true. Will update.